core/dictgen/src/BaseSelectionRule.cxx

```diff
@@ -317,7 +317,7 @@ BaseSelectionRule::EMatchType BaseSelectionRule::Match(const clang::NamedDecl *d
          if (!patternMatched &&
                D &&
                //ROOT::TMetaUtils::IsStdDropDefaultClass(*D)) {
-               0 != TClassEdit::IsSTLCont(name)) {
+               0 != TClassEdit::IsSTLCont(std::string_view(name.data(), name.size()))) {
```

Comment:
Why do we need to construct a temporary `string_view` here, explicitly, from a `std::string`?